src/RadhydroSimulation.hpp

```diff
@@ -139,6 +141,7 @@ template <typename problem_t> class RadhydroSimulation : public AMRSimulation<pr
 	amrex::Long radiationCellUpdates_ = 0; // total number of radiation cell-updates
 
 	amrex::Real Gconst_ = C::Gconst; // gravitational constant G
+	std::unique_ptr<Gravity<problem_t>> gravitySolver;
```

Comment:
I created the gravity solver object here, but I apparently never initialized it. In the test problem in `test_poisson.cpp`, I create a separate `Gravity<problem_t>` object just for testing purposes.

Reply:
So do I remove this line?

Reply:
This is needed in either `AMRSimulation` or `RadhydroSimulation`. Which one is up to you.

Reply:
Okay lets keep it here as it is then.